src/index.html

```diff
@@ -7,8 +7,170 @@
+              <div id="educ_P2" class="educ_P"></div>
+              <div id="educ_T3" class="educ_T"></div>
+              <div id="educ_time3" class="educ_time"></div>
+              <div id="educ_P3" class="educ_P"></div>
+            </div>
+          </li>
+          <li>
+            <div id="hobb_section">
+              <div id="hobb_t">HOBBIES&INTERESTS</div>
+              <div id="hobb_down">DOWNLOAD MY FULL RESUME</div>
+              <hr />
+              <img src="" alt="hobby_1" class="hobb_pic" />
+              <img src="" alt="hobby_2" class="hobb_pic" />
+              <img src="" alt="hobby_3" class="hobb_pic" />
+              <img src="" alt="hobby_4" class="hobb_pic" />
+              <img src="" alt="hobby_5" class="hobb_pic" />
+              <img src="" alt="hobby_6" class="hobb_pic" />
+            </div>
+          </li>
+        </ul>
+      </div>
+
+      <div id="skls">
+        <div id="skls_t">SKILLS</div>
+        <hr />
+        <div id="skls_map">
+          <div id="skls_1" class="skls_total">
+            <div id="skilname_1" class="skilname">Microsoft</div>
+            <div id="skilprof_1" class="skilprof"></div>
+          </div>
+          <div id="skls_2" class="skls_total">
+            <div id="skilname_2" class="skilname">Microsoft</div>
+            <div id="skilprof_2" class="skilprof"></div>
+          </div>
+          <div id="skls_3" class="skls_total">
+            <div id="skilname_3" class="skilname">Microsoft</div>
+            <div id="skilprof_3" class="skilprof"></div>
+          </div>
+          <div id="skls_4" class="skls_total">
+            <div id="skilname_4" class="skilname">Microsoft</div>
+            <div id="skilprof_4" class="skilprof"></div>
+          </div>
+          <div id="skls_5" class="skls_total">
+            <div id="skilname_5" class="skilname">Microsoft</div>
+            <div id="skilprof_5" class="skilprof"></div>
+          </div>
+          <div id="skls_6" class="skls_total">
+            <div id="skilname_6" class="skilname">Microsoft</div>
+            <div id="skilprof_6" class="skilprof"></div>
+          </div>
+        </div>
+      </div>
+      <div id="conta">
+        <div id="quicid">
+          <div id="quicid_t">QUICK ID</div>
+          <hr />
+          <div id="quicid_n1" class="quicid_name"></div>
+          <div id="quicid_d1" class="quicid_detial"></div>
+          <div id="quicid_n2" class="quicid_name"></div>
+          <div id="quicid_d2" class="quicid_detial"></div>
+          <div id="quicid_n3" class="quicid_name"></div>
+          <div id="quicid_d3" class="quicid_detial"></div>
+          <div id="quicid_n4" class="quicid_name"></div>
+          <div id="quicid_d4" class="quicid_detial"></div>
+        </div>
+        <div id="contame">
+          <div id="quicid_t">QUICK ID</div>
+          <hr />
+          <form>
+            <input type="text" name="name" value="Name" />
```

Comment:
Please set all the `value` attributes to "" and add in `placeholder`.
It is always good to include labels due to accessibility concern.

Reply:
It's a mistake which I use value attributes,I planned to use placeholders to indicate which information should put in input boxes since there are no labels on the website.
I have a question about the last sentence you pointed out.
will there be a conflict or problem if not use the label to indicate the input box, because I have seen many websites don't use labels?

Reply:
This is something we will briefly go through on lesson 2.

There is no right or wrong to not include a label for each field, but it is highly recommended to include one for accessibility and usability reasons. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

You can hide it with CSS if you don't want the label visible for stying reasons. You can make the label "placeholder" like with the power of CSS and JS. (e.g. https://css-tricks.com/float-labels-css/)

It is considered an usability issue to only have placeholder attribute too since once you filled the field content in, you have no title for that field anymore.

There are extensive studies on form usability, but it is not with our scope in this course. You can read this article if you interested: https://www.smashingmagazine.com/2011/11/extensive-guide-web-form-usability/